saleor/graphql/order/mutations/orders.py

```diff
@@ -414,3 +418,35 @@ def perform_mutation(cls, _root, info, amount, **data):
 
         order_refunded(order, info.context.user, amount, payment)
         return OrderRefund(order=order)
+
+
+class OrderUpdateMeta(UpdateMetaBaseMutation):
+    class Meta:
+        description = "Updates meta for order."
+        model = models.Order
+        permissions = "orders.manage_orders"
+        public = True
+
+
+class OrderUpdatePrivateMeta(UpdateMetaBaseMutation):
+    class Meta:
+        description = "Updates private meta for order."
+        model = models.Order
+        permissions = "orders.manage_orders"
+        public = False
+
+
+class OrderClearStoredMeta(ClearMetaBaseMutation):
```

Comment:
We have a few meta-related mutations that use the word `Stored` which seems redundant. It would be much simpler to read and it would have the same meaning if it was named `OrderClearMeta`. @szewczykmira Could you look through all meta mutations and simplify those names? It wasn't officially released yet so I think we can change it without deprecation.

Reply:
@maarcingebala I've updated those mutations. I've also updated their names in schema - let me know if that's okey

Reply:
Thanks @szewczykmira, it looks good now!